Loop/Managers/LoopDataManager.swift

```diff
@@ -607,7 +552,7 @@ extension LoopDataManager {
         }
         self.dosingDecisionStore.storeDosingDecision(dosingDecision) {}
 
-        self.notify(forChange: .tempBasal)
+        self.notify(forChange: .loopFinished)
```

Comment:
I'll change this back; this rename happened for an earlier version of updating forecast/manual bolus amount for upload to remote caregivers after a loop runs.

Reply:
On second thought. tempBasal is no longer accurate either, since we do boluses as well. In the case of `cancelActiveTempBasal`, a full loop didn't run, so "loopFinished" is not great either. But the part of Loop that does automatic dosing (LDM) has made another decision, similar to a normal loop. Going to keep 'loopFinished' here. Will add a comment.